### PR TITLE
feat(replays): add replays rage click issue creation toggle

### PR DESCRIPTION
**Pull request by JoshFerge**
- [x] creates replay subsection on project settings, under 'processing'
- [x] adds a toggle to control a project option for rage click issue creation

Depends on creating project option that the toggle hooks up to. creating PR now so we can iterate on copy. Also need to start thinking about what the docs section for rage click issues could look like. for now just links to [here](https://docs.sentry.io/product/session-replay/replay-page-and-filters/) which mentions rage clicks. 
Screenshot 2024-01-30 at 3 10 19 PM

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
A new replay section is consistent with what we do with e.g. Performance issues, but wondering if it's necessary since we only have one option. Though assuming we add more in the future, this is better than shuffling things around in the future.

File: static/app/views/settings/project/navigationConfiguration.tsx [hunk 1]
```diff
@@ -121,6 +121,11 @@ export default function getConfiguration({
               organization?.features?.includes('ddm-ui')
             ),
         },
+        {
+          path: `${pathPrefix}/replays/`,
+          title: t('Replays'),
+          show: () => !!organization?.features?.includes('session-replay-ui'),
```
[the remark this answers is not in the document]
Comment (JoshFerge) [reply static/app/views/settings/project/navigationConfiguration.tsx:127]: yes, for now anyways

[the remark this answers is not in the document]
Comment (JoshFerge) [reply static/app/views/settings/project/navigationConfiguration.tsx:127]: and @billyvg  yeah we only have one now, but i could see us adding more in the future

File: static/app/views/settings/project/projectReplays.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  }
+}
+
+export default ProjectUserFeedbackSettings;
+
+const ButtonList = styled('div')`
+  display: inline-grid;
+  grid-auto-flow: column;
+  gap: ${space(1)};
+`;
```
Comment (billyvg) [inline static/app/views/settings/project/projectReplays.tsx:80]: Is this necessary since there's only one child?

**Review by billyvg — APPROVED** [review]

File: static/app/views/settings/project/projectReplays.tsx (hunk 2 above)
Comment (JoshFerge) [reply static/app/views/settings/project/projectReplays.tsx:80]: idk was just copying from the feedback settings page

**Pull request merged**